[#12444] Updated development.md (#12525)
TEAMMATES/teammates: docs/development.md

If you have access to Docker, we have a Docker compose definition to run those services:

```sh
docker compose up -d

```

> If the following command does not work, you may not have the updated version (v2). So use this instead:

Reviewer: ```suggestion
If the above command does not work, you may not have the updated version (v2) of Docker. You may want to try this instead:
```

Reviewer: To me, "... following command..." seems to refer to the command **after** this sentence (i.e. `docker-compose`), which seems out of order. I think it would be clearer to use "... above command..."

```sh
docker-compose up -d
```

> For more information refer to the [Docker Documentation](https://docs.docker.com/compose/reference/)

Reviewer: ```suggestion
For more information on Docker, you may wish to refer to the [Docker Documentation](https://docs.docker.com/compose/reference/).
```

We have a Docker compose definition to run dependent services, including local Datastore emulator. Run it under the `datastore` service name and bind to the container port `8484`:

```sh
docker compose run -p 8484:8484 datastore
```

> If the above command does not work, you can use this command:
> 

Reviewer 2: ```suggestion
If the above command does not work, you may want to try this instead:

```
